File: 1950A/1950A.py
n = int(input())
x=[]
for i in range(n):
    a , b, c = map(int , input().split())
    if a < b and b < c: x.append("STAIR")
    elif a <b and b > c : x.append("PEAK")
    else: x.append("NONE")
for i in x:
    print(i)

File: 1950A/1950A.cpp
#include<iostream>
#include<vector>
using namespace std;
int main()
{
    int n;
    cin>>n;
    vector<string> v(n);
    for(int i=0;i<n;i++)
    {
        int a , b , c;
        cin >> a >> b >> c;
        if (a < b and b < c) v[i] = "STAIR";
        else if ( a < b and b > c ) v[i] = "PEAK";
        else v[i] = "NONE";
    }
    for(auto& i : v){
        cout << i << endl;
    }
    return 0;
}